### ISSUE TITLE
E15/E1 - Inconsistent Identification

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#147]
The 'Filtered by Date' sidebar on the events listing page for Upcoming Events is identified by the heading "Events Archive", whereas the same sidebar on the page for Past Events is identified by the heading "Browse Events by Date". Different heading text is used to describe the same component, failing criterion 3.2.4.

Upcoming Events
![upcoming events](https://cloud.githubusercontent.com/assets/12450480/9250841/fc503d32-419d-11e5-9455-b69551c10890.PNG)
Past Events
![past events](https://cloud.githubusercontent.com/assets/12450480/9250834/f8691194-419d-11e5-9adc-1d1f60525717.PNG)

**Comment by jpinsent** [comment]
This appears to have been resolved as a result of the hotfix for issue #91
